scripts/validate.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 REPO_ROOT=$(git rev-parse --show-toplevel)
-SCHEMA_FILE="${REPO_ROOT}/scripts/io.mediachain.indexer_image_jsonschema_1-0-0.json"
+SCHEMA_FILE="${REPO_ROOT}/scripts/io.mediachain.indexer-image-jsonschema-1-0-0.json"
```

Comment:
ok am I weird for preferring `package-name-with-stuff_1-0-0.json`? i.e. underscore as name/version separator?